### ISSUE TITLE
Glob NullPointerException

### ISSUE THREAD

**Issue by rnveach** [hcoles/pitest#539]
Accidentally adding a empty `TargetTest` results in an NPE.

Example:
````
              <targetTests>
                <param>com.puppycrawl.tools.checkstyle.xpath.*</param>
                <param>com.puppycrawl.tools.checkstyle.XpathFileGeneratorAuditListenerTest</param>
                <param>com.puppycrawl.tools.checkstyle.XpathFileGeneratorAstFilterTest</param>
                <param></param>
              </targetTests>
````

Trace:
````
[ERROR] Failed to execute goal org.pitest:pitest-maven:1.4.3:mutationCoverage (default-cli) on project checkstyle: Execution default-cli of goal org.pitest:pitest-maven:1.4.3:mutationCoverage failed. NullPointerException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.pitest:pitest-maven:1.4.3:mutationCoverage (default-cli) on project checkstyle: Execution default-cli of goal org.pitest:pitest-maven:1.4.3:mutationCoverage failed.
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:224)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:120)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:355)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:155)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:584)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:216)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:160)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution default-cli of goal org.pitest:pitest-maven:1.4.3:mutationCoverage failed.
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:143)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	... 19 more
Caused by: java.lang.NullPointerException
	at org.pitest.util.Glob.<init>(Glob.java:29)
	at org.pitest.util.Glob.lambda$toGlobPredicate$0(Glob.java:42)
	at org.pitest.functional.FCollection.mapTo(FCollection.java:42)
	at org.pitest.functional.FCollection.map(FCollection.java:50)
	at org.pitest.maven.MojoToReportOptionsConverter.determineTargetTests(MojoToReportOptionsConverter.java:237)
	at org.pitest.maven.MojoToReportOptionsConverter.parseReportOptions(MojoToReportOptionsConverter.java:103)
	at org.pitest.maven.MojoToReportOptionsConverter.convert(MojoToReportOptionsConverter.java:81)
	at org.pitest.maven.AbstractPitMojo.analyse(AbstractPitMojo.java:454)
	at org.pitest.maven.AbstractPitMojo.execute(AbstractPitMojo.java:396)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:132)
	... 20 more
````

Obviously this is my fault, but pitest could handle this issue more gracefully like display an error or ignore the empty line.

**Comment by maxgabut** [comment]
Thanks for the report. I think that ignoring the empty value is the way to go.

**Comment by maxgabut** [comment]
Fixed for versions > 1.4.5
